# awesome-microbes
List of software packages (and the people developing these methods) for microbiome (16S), metagenomics (WGS, Shot-gun sequencing), and pathogen identification/detection/characterization.  [Contributions welcome...](https://github.com/stevetsa/awesome-microbes/blob/master/CONTRIBUTE.md)
<br /> 
Inspired by [awesome-single-cell](https://github.com/seandavi/awesome-single-cell/blob/master/README.md)
<br />


## Microbiome (16S)

[Qiime](http://qiime.org/) - [Python] - QIIME is designed to take users from raw sequencing data generated on the Illumina or other platforms through publication quality graphics and statistics. This includes demultiplexing and quality filtering, OTU picking, taxonomic assignment, and phylogenetic reconstruction, and diversity analyses and visualizations.

[mothur](https://www.mothur.org/) - [C++] - OTU-based analysis of 16S data.


## Metagenomics (WGS, Shot-gun sequencing)

[bioBakery-MetaPhlAn](https://bitbucket.org/biobakery/biobakery/wiki/Home) - [python] - a virtual environment platform that provides meta'omic analysis tools.


## Microbe (pathogen, bacterial, viral) Identification/Detection/Characterization

[Pathoscope](https://sourceforge.net/p/pathoscope/wiki/Home/) - [python] - Species identification and strain attribution with unassembled sequencing data.
